### ISSUE TITLE
Controlling the default output channel

### ISSUE THREAD

**Issue by keybounce** [jackaudio/jack2#100]
I am using an aggregate device with Jack. Jack wants to send all output by default to channels 1 and 2.

I cannot control which device the aggregate puts into the first slot.
Is there any way to tell jack which channels are the default output channels?

As an example of what I'd like to be able to do, I'd like to say "Map audio 1 to output 4, map audio 2 to output 2". That would put the left audio on my left external speaker, and the right audio on my right internal speaker.

**Comment by mseeber** [comment]
I am using an aggregate device
So you are on a MAC?

**Comment by keybounce** [comment]
Yes. As I just said in #99, I thought this was reporting issues on the Mac port, not the general development. (My fault).
